deploy/kubernetes/helm/che/custom-charts/che-keycloak/templates/deployment.yaml

```diff
@@ -54,10 +54,23 @@ spec:
           value: keycloak
         - name: DB_PASSWORD
           value: keycloak
+{{- if .Values.keycloakAdminUserSecret }}
         - name: KEYCLOAK_USER
-          value: {{ .Values.keycloakAdminUserName }}
+          valueFrom:
+            secretKeyRef:
+              name: {{ .Values.keycloakAdminUserSecret }}
+              key: KEYCLOAK_USER
         - name: KEYCLOAK_PASSWORD
-          value: {{ .Values.keycloakAdminUserPassword }}
+          valueFrom:
+            secretKeyRef:
+              name: {{ .Values.keycloakAdminUserSecret }}
+              key: KEYCLOAK_PASSWORD
+{{- else }}
+        - name: KEYCLOAK_USER
+          value: {{ .Values.keycloakAdminUserName | default "admin" }}
```

Comment:
I am not sure if we need default since we have it in `values.yaml`

Reply:
You are right, but I thought some defensive programming wouldn't harm.  I find sometimes it acts as unofficial documentation!  But I am fine with removing the default if you want to keep it consistent with the rest of the chart.